tiledb/vfs.py

```diff
@@ -304,6 +304,11 @@ def touch(self, uri: _AnyPath):
         """
         return self._touch(_to_path_str(uri))
 
+    # Aliases for compatibility with fsspec's AbstractFileSystem.
+    isdir = is_dir
+    isfile = is_file
+    size = file_size
```

Comment:
That's all it takes to make the tests green. If you `help(isdir)` you get the docstring from `is_dir`, but that's not terrible.